src/reason_pprint_ast.ml

```diff
@@ -3014,6 +3011,23 @@ class printer  ()= object(self:'self)
         in
         let rightItm = self#simplifyUnparseExpr rightExpr in
         Some (label ~space:forceSpace (atom prefixStr) rightItm)
+      | (Infix s, [(_, e1); (_, e2)]) when s.[0] = '#' ->
```

Comment:
Are you saying the naming / AST node is wrong? I just tried and `add x##y z##q` parses and prints correctly.